### ISSUE TITLE
Classic Block Toolbar Not Keyboard Accessible

### ISSUE THREAD

**Issue by timwright12** [WordPress/gutenberg#6053]
## Issue Overview
Can't use classic block toolbar with keyboard (without a mouse)

## Steps to Reproduce (for bugs)
1. Add a classic block
2. Try to bold text without a mouse

## Expected Behavior
Like in the normal block, you should be able to use the toolbar without a mouse.

## Current Behavior
Focus leaves the block, skipping over the toolbar.

**Comment by karmatosed** [comment]
Closing as similar to #6118.
